src/main/java/org/example/DBController.java

```diff
@@ -128,6 +129,142 @@ public boolean updateCustomerFunds(int customerId, int newAmount) {  //void
         return false;
     }
 
+    public void createTransactionAudit(int senderId, String senderName, int moneyInCents, int getterId, String getterName) throws SQLException {
+        String sql = "INSERT INTO transactions_audit (sender_id, sender_nickname, amount_sent, receiver_id, receiver_nickname) " + "VALUES (?, ?, ?, ?, ?)";
+
+        PreparedStatement ps = connection.prepareStatement(sql,
+                Statement.RETURN_GENERATED_KEYS);

```

Comment:
See how the "correct" multiline parameters should look:
```suggestion
        PreparedStatement preparedStatement = connection.prepareStatement(
            sql,
            Statement.RETURN_GENERATED_KEYS
        );
```
This is up for debate, but is generally easier to read. I noticed a lot of Java style guides tend to be fairly old. Since you're using Intellij IDEA I suggest setting autoformatting and letting the IDE format your files for you. See https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat-on-save.

Also see https://www.jetbrains.com/help/idea/code-style-java.html. You can tweak these rules, but the defaults are fairly sensible as-is.